cloudfour/cloudfour.com-patterns
# Chore: add SVG favicon with dark mode support

## Overview

This PR represents a single task as part of #746, which adds a favicon in SVG format and experiments with adding support for dark mode via `prefers-color-scheme` media queries. 

## Screenshots

After reading [this article](https://css-tricks.com/svg-favicons-and-all-the-fun-things-we-can-do-with-them/#keep-it-crisp), I thought I would make an attempt at decreasing antialiasing. This resulted in the the size of the favicon being slightly reduced. I'm not sure if this was the correct decision and therefore am requesting feedback. 

<img width="1682" alt="Screen Shot 2020-08-17 at 3 12 28 PM" src="https://user-images.githubusercontent.com/1427548/90449358-1c261800-e09c-11ea-816a-32399458cb8d.png">

<img width="1252" alt="Screen Shot 2020-08-17 at 1 44 06 PM" src="https://user-images.githubusercontent.com/1427548/90442647-d82d1600-e08f-11ea-882e-dd1a6b77c662.png">

I also thought I would experiment with adding support for dark mode via `prefers-color-scheme`:

<img width="1571" alt="Screen Shot 2020-08-17 at 12 43 34 PM" src="https://user-images.githubusercontent.com/1427548/90437494-589b4900-e087-11ea-83f1-0a42ddad4bf5.png">

<img width="1570" alt="Screen Shot 2020-08-17 at 12 43 42 PM" src="https://user-images.githubusercontent.com/1427548/90437503-5cc76680-e087-11ea-91ee-31e76efc26f5.png">

Since the "dark mode" favicon's fill color is white, and is presented on a white backdrop in Storybook, I added a `<style>` block with prefers-color-scheme media queries to the twig story file that will also toggle its color, which makes the white favicon visible when dark mode is selected.

## Testing

1. View [deploy preview](https://deploy-preview-908--cloudfour-patterns.netlify.app/?path=/docs/design-brand--favicon)
1. Provide feedback on size of favicon
1. Toggle dark mode on your system and confirm prefers-color-scheme works as expected
1. Provide feedback on the decision to add dark mode support and the color selections presented in this PR
1. If appropriate, critique and provide feedback for how I modified the story backdrop color to demonstrate the dark mode favicon
1. Provide code review

---

see: issue #746
